src/gsi/read_anowbufr.f90

```diff
@@ -198,9 +228,10 @@ subroutine read_anowbufr(nread,ndata,nodata,gstime,&
         
         nread = nread + 1
         conc=indata(ncopopm)
-        
         if ( iret > 0 .and. (conc < conc_missing ) .and. &
                (conc >= zero)) then
+           if(indata(nxob) > r360)cycle 
+           if(indata(nyob) > 0.5_r_kind*r360)cycle
```

Comment:
why don't directly use r180 (if r180 is not defined,  it could be defined along with r360).  The extra r180 will avoid the multiplication for 0.5*r360 for each obs.